test/functional/tests/cli/test_cli_script.py

```diff
@@ -0,0 +1,80 @@
+#
+# Copyright(c) 2020 Intel Corporation
+# SPDX-License-Identifier: BSD-3-Clause-Clear
+#
+
+
+import pytest
+
+from api.cas import casadm, casadm_parser
+from core.test_run import TestRun
+from test_utils.os_utils import sync
+from storage_devices.disk import DiskType, DiskTypeSet, DiskTypeLowerThan
+from test_utils.size import Unit, Size
+from test_tools.dd import Dd
+
+
+@pytest.mark.require_disk("cache", DiskTypeSet([DiskType.nand, DiskType.optane]))
+@pytest.mark.require_disk("core", DiskTypeLowerThan("cache"))
+@pytest.mark.parametrize("purge_target", ["cache", "core"])
+def test_purge_cache(purge_target):
```

Comment:
In this test You call two commands, not only `purge cache` like the function name says.